docs/usage/templates.md

```diff
@@ -49,6 +49,16 @@ In the example above `baseDir` is the string you want to transform into a valid
 
 Read the [MDN Web Docs, encodeURIComponent()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) to learn more.
 
+### decodeURIComponent
+
+If you want to decode a percent-encoded string, use the built-in function `decodeURIComponent` like this:
+
+`{{{decodeURIComponent depName}}}`
```

Comment:
Is it definitely a triple brace and not double?

Reply:
the other helpers are defined in same way